types/src/dkg/dkg_committee.rs

```diff
@@ -0,0 +1,24 @@
+use move_core_types::account_address::AccountAddress;
+use serde::{Deserialize, Serialize};
+
+/// Reflection of `0x1::types::DkgCommitteeType` in rust.
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+pub enum DkgCommitteeType {
+    Clan,
+    Tribe,
+}
+
+/// Reflection of `0x1::dkg_committee::DkgNodeConfig` in rust.
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+pub struct DkgNodeConfig {
+    pub addr: AccountAddress,
```

Comment:
It would be helpful to be more descriptive here.
```suggestion
    pub pool_address: AccountAddress,
```

Reply:
fixed